src/Network/Pusher/Internal.hs

```diff
@@ -87,6 +89,17 @@ mkUsersRequest pusher chan time =
   let subPath = "channels/" <> renderChannel chan <> "/users"
   in mkGetRequest pusher subPath [] time
 
+mkNotifyRequest :: Pusher
+                -> Notification
+                -> Int
+                -> Either PusherError (RequestParams, RequestBody)
+mkNotifyRequest pusher notification time = do
+  let body = A.toJSON notification
+      bodyBS = BL.toStrict $ A.encode body
+  when (B.length bodyBS > 10000) $
+    Left $ PusherArgumentError "Body must be less than 10000KB"
+  return $ (mkNotifyPostRequest pusher "notifications" [] bodyBS time, body)
```

Comment:
Can you add a couple of tests for this following a similar pattern to the existing ones? You could find some examples from the docs and check the returned JSON is what you'd expect.